test/test_vmap.py

```diff
@@ -2698,12 +2701,12 @@ def test_conv2d(self):
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(conv_fn, arg_values, kwarg_values):
                 self.assertEqual(loop_out, batched_out)
 
-            mod2 = torch.nn.Conv2d(4, 8, kernel_size=3, groups=2, stride=3, padding=1, dilation = 2)
-            arg_values = [torch.randn(inp_shape), mod.weight, mod.bias]
+            mod2 = torch.nn.Conv2d(4, 8, kernel_size=3, groups=2, stride=3, padding=1, dilation=2)
```

Comment:
Okay from the test failure, let's try having this be `conv_mod` instead of `torch.nn.Conv2d`? We can also disable this part of the test and file an issue if this is blocking

Reply:
Oh, I see. I replaced a bit blindly unused variable: `mod -> mod2`. Thanks for pointing out @samdow !